andela/travel_tool_back
# #160820728 Email notification for edited travel requests

#### What does this PR do?
This PR adds email notification for updated travel request

#### Description of Task to be completed?
Modify the notification manager
Add update message to email template for updated requests

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_back.git`
- Enter into the project directory - `cd travel_tool_back`
- Rollback and run database migrations - `yarn db:rollmigrate`
- Add `APP_EMAIL`, `REACT_APP_AUTH_REDIRECT_URL`, `SENDGRID_API_KEY` to `.env` file
- Install dependencies and start the server - `make start`
-  [Start](https://github.com/andela/travel_tool_front/pull/107)  the frontend.
- Make a new request and choose yourself as the manager
- update the created request
- Check your andela email account for a new message from travela regarding an updated request
- Click on the `view request` button on the email to redirect to the request

#### Any background context you want to provide?
- In other to test this feature, you need to update your role on the Users table to that of a manager.
Manager roleId: `50319`

#### What are the relevant pivotal tracker stories?
* [#160820728](https://www.pivotaltracker.com/story/show/160820728)

#### Screenshots (if appropriate)
<img width="1082" alt="screen shot 2018-10-04 at 9 54 43 am" src="https://user-images.githubusercontent.com/32720508/46466201-8d1ef800-c7c2-11e8-9d50-425cb939a8d7.png">

#### Questions:
N/A
